cpp/benchmarks/common/generate_input.cu

```diff
@@ -792,6 +792,16 @@ std::unique_ptr<cudf::table> create_random_table(std::vector<cudf::type_id> cons
   return std::make_unique<cudf::table>(std::move(output_columns));
 }
 
+std::unique_ptr<cudf::column> create_random_column(cudf::type_id dtype_id,
```

Comment:
I am somewhat saddened by the fact that `create_random_table` doesn't call `create_random_column`, but that would require some changes to prevent seeding the random number generator for each column.

Reply:
done!